.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```

Comment:
```suggestion
**Expected behavior**
A clear and concise description of what you expected to happen.

Expected output if applicable:
``` text
expected output here
```
```

Reply:
The above also seems to be missing the last 3 backticks.

Reply:
The above suggestion looks like so:

**Expected behavior**
A clear and concise description of what you expected to happen.

Expected output if applicable:
``` text
expected output here
```